react-jsx/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "react-jsx",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build":"webpack",
+    "watch": "webpack --watch --mode=development --devtool=source-map",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-loader": "^8.2.3",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2"
+  },
+  "dependencies": {
+    "@babel/plugin-transform-react-jsx": "^7.16.7",
```

Comment:
This one should be under `devDependencies`, only `react` and `react-dom` are `dependencies`